generator/internal/tracing_stub_generator.cc

```diff
@@ -241,14 +247,28 @@ future<StatusOr<google::longrunning::Operation>>
     google::cloud::CompletionQueue& cq,
     std::shared_ptr<grpc::ClientContext> context,
     google::longrunning::GetOperationRequest const& request) {
-  return child_->AsyncGetOperation(cq, std::move(context), request);
+  auto span =
+      internal::MakeSpanGrpc("google.longrunning.Operations", "GetOperation");
```

Comment:
Opt: This and the next `auto span = ...` assignments are reflowed appropriately when formatting the generated libraries, but would it be nice to make them similarly wrapped within these templates?

Reply:
I did notice this. I think I'd rather get out of the business of thinking about how to format the code, even when it's in these raw strings.